_pages/en_US/uninstall-cfw.txt

```diff
@@ -6,65 +6,128 @@ title: "Uninstall CFW"
+{: .notice--danger}  
+
+#### Section IV - NAND Backup
+
+1. Launch GodMode9 by holding (Start) during boot
+1. Press (Home) to bring up the action menu
+1. Select "Scritps..."
+1. Select "GM9Megascript"
+1. Select "Backup Options"
+1. Select "SysNAND Backup"
+1. Press (A) to confirm
+  + This process will take some time
+  + If you get an error, ensure you have at least 1.3GB of free space on your SD card
+1. Press (B) to return to the main menu
+1. Select “Exit”
+1. Press (Home) to bring up the action menu
+1. Select “Poweroff system” to power off your device
+
+#### Section V - Removing illegitimate content
+This section will remove illegitimate content, like homebrew and dumped cartridges. If you have save data that you care about, back it up with a save manager before continuing!
+{: .notice--warning}
 
 1. Power on your device
-1. Navigate to System Settings > Data Management > Nintendo 3DS > Software
+1. Open the System Settings application
+1. Navigate to Data Management > Nintendo 3DS > Software
 1. In this list of software, delete any non-Nintendo content you installed while using CFW
-  + This includes common system software such as FBI, Anemone3DS, Luma Updater, Homebrew Launcher, Checkpoint, and others, along with any non-eShop games
+  + This includes common system software such as FBI, Anemone3DS, Luma Updater, Homebrew Launcher, Checkpoint, and others, along with any games and titles that you did *not* install from the eShop
 1. Navigate to Data Management > DSiWare
 1. In this list of software, delete any non-Nintendo content you installed while using CFW
-  + This includes software such as TWiLightMenu++, along with any non-eShop games
+  + This includes software such as TWiLightMenu++, along with any games and titles that you did *not* install from the eShop
   + Failure to remove all CFW software from both the 3DS and DSiWare sections before uninstalling CFW may prevent or disable access to the Data Management menu after uninstalling CFW
-1. Power off your device
+1. Exit the System Settings application
+1. Open the Homebrew Launcher application
+1. Launch DSiWare Uninstaller from the list of homebrew
+1. Follow the prompts and allow the program to uninstall 
+1. Once the process has succeeded, exit the Homebrew Launcher and power off your device
+
+#### Section VI - System Format
+This section will ensure that all illegitimate tickets are removed, allowing eShop to work normally. This will remove all content from the 3DS and log you out of your NNID. Keep in mind that your console's encryption key will be shuffled, meaning that any old data will be rendered inaccessible, even if you have a backup of your SD contents.
 
-#### Section III - Running Uninstall Script
+1. Power on your device
+1. Open the System Settings application
+1. Navigate to Other Settings -> Next Page (four times) -> Format System Memory
```

Comment:
I think that on certain 3DS models there's only 4 pages of "Other settings", and not 5? I might be wrong on this though

Reply:
JPN/USA N3DS XL have five pages, USA O3DS XL has four. Will amend